code/messaging/src/main/java/com/adobe/marketing/mobile/MessagingPushPayload.java

```diff
@@ -212,6 +213,25 @@ private int getNotificationPriorityFromString(final String priority) {
         }
     }
 
+
+    @RequiresApi(api = Build.VERSION_CODES.N)
+    private int getNotificationImportanceFromString(final String priority) {
+        if (priority == null) return Notification.PRIORITY_DEFAULT;
+            switch (priority) {
+                case NotificationPriorities.PRIORITY_MIN:
+                    return NotificationManager.IMPORTANCE_MIN;
+                case NotificationPriorities.PRIORITY_LOW:
+                    return NotificationManager.IMPORTANCE_LOW;
+                case NotificationPriorities.PRIORITY_HIGH:
+                    return NotificationManager.IMPORTANCE_HIGH;
+                case NotificationPriorities.PRIORITY_MAX:
+                    return NotificationManager.IMPORTANCE_MAX;
+                case NotificationPriorities.PRIORITY_DEFAULT:
+                default:
+                    return NotificationManager.IMPORTANCE_NONE;
```

Comment:
maybe we could use default importance instead? `NotificationManager.IMPORTANCE_DEFAULT`

Reply:
that sounds right. Thanks for the explanation